types/src/crypto/hpke.rs

```diff
@@ -42,6 +42,16 @@ use super::{
 #[sqlx(transparent)]
 pub struct EncryptionPublicKey(Vec<u8>);
 
+impl EncryptionPublicKey {
+    pub(crate) fn from_bytes(value: Vec<u8>) -> Self {
+        Self(value)
+    }
+
+    pub(crate) fn into_bytes(self) -> Vec<u8> {
+        self.0
+    }
+}
```

Comment:
After merging `main`, you can make this into a marker trait and implement it only for the keys that need it.

Reply:
Added a marker trait `RawKey` s.t. `Key<KT: RawKey>` implements public `from_bytes/into_bytes`.